src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ValidationConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.config;
+
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.Validator;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.Consumer;
+
+@Configuration
+public class ValidationConfiguration {
+
+    @Bean
+    public Validator jsonSchemaValidator() {
+        return Validator.builder().failEarly().build();
```

Comment:
shall we not gather all errors instead?

Reply:
Even I would prefer gathering all errors(if possible) as it becomes easier when dealing with Exela that which all fields were invalid.